contents/blog/best-mobile-app-ab-testing-tools.mdx

```diff
@@ -0,0 +1,315 @@
+---
+date: 2023-11-21
+title: The 10 best mobile app A/B testing tools
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/open-source-testing-tools/testinghog.png
+featuredImageType: full
+author:
+  - lior-neu-ner
+category: General
+tags:
+  - Guides
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+Also known as split testing or [multivariate testing](/product-engineers/what-is-multivariate-testing-examples), A/B testing enables you to measure the impact of product changes on your mobile apps.
+
+There are many A/B testing tools out there, but each one is built for a specific target audience in mind. In this guide, we’ll explore these options so that you can decide which is best for you.
+
+## 1. PostHog
+
+![PostHog - best ab testing tools](../images/blog/best-mobile-app-ab-testing-tools/posthog.png)
+
+- **Best for:** Engineering and product teams
+- **Free tier:** <span className="text-green text-lg">✔</span>
+- **Transparent pricing:** <span className="text-green text-lg">✔</span>
+- **Open source:** <span className="text-green text-lg">✔</span>
+- **Get started without talking to sales:** <span className="text-green text-lg">✔</span>
+- **Includes product analytics:** <span className="text-green text-lg">✔</span>
```

Comment:
Originally I had this as a table in each section, but I opted to change it to this as I think it makes it easier to read.

The only bit I'm a bit unsure of is the `Includes product analytics`. What Im trying to say is that for some of these app also provide product analytics so you dont need a separate platform to get a full picture of your A/B test results.

Reply:
This looks better.